src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -13,7 +13,7 @@ import scala.annotation.tailrec
 
 object Import {
 
-  case class Endpoint(protocol: String, bindPort: Int, services: Set[URI])
+  case class Endpoint(protocol: String, bindPort: Int = 0, services: Set[URI] = Set.empty)
```

Comment:
+docs about the 0 value? It would have helped resolve my confusion when using this the first time.